users/tasks.py

```diff
@@ -0,0 +1,16 @@
+from celery import shared_task
+from django.utils import timezone
+from datetime import timedelta
+from users.models import SchoolUser
+
+
+@shared_task
+def block_inactive_users():
+    """Блокировка пользователей, не заходивших более месяца"""
+    month_ago = timezone.now() - timedelta(days=30)
```

Comment:
Можно задать чуть точнее
```suggestion
    month_ago = timezone.now() - relativedelta(months=1)
```